spec/lib/facilities/ppms/v0/client_spec.rb

```diff
@@ -53,7 +53,7 @@
           Latitude: 33.262403,
           Longitude: -111.808538,
           MainPhone: nil,
-          Miles: 1.679,
+          Miles: 1.842,
```

Comment:
What's the reason for this value change?

Reply:
PPMS returned a different result for the same query.